app/server/models/transfer_account.py

```diff
@@ -111,18 +117,13 @@ def balance(self):
 
     @balance.setter
     def balance(self, val):
-        self._balance_wei = val * int(1e16)
+        self._balance_offset = val * int(1e16)
```

Comment:
I don't think this should be under the balance setter, because it's _not_ actually setting the balance.

Suggest rather just a plan 'ol method called `set_balance_offset` or something, and leave the balance setter undefined as it's not really allowed unless in tests